EasyLM/data.py

```diff
@@ -268,6 +280,66 @@ def vocab_size(self):
         return len(self._tokenizer)
 
 
+class HFPromptDataset(object):
+    @staticmethod
+    def get_default_config(updates=None):
+        config = ConfigDict()
+        config.path = 'openbmb/UltraFeedback'
+        config.name = ''
+        config.split = 'train'
+        config.streaming = False
+        config.seq_length = 1024
+        config.num_workers = 8
+        config.batch_size = 8
+        config.policy_prefix_tokens = '<|user|>\n'
+        config.policy_suffix_tokens = '\n<|assistant|>\n'
+        config.reward_prefix_tokens = 'Human: '
```

Comment:
The current RMs I train expect tulu format, maybe we should swap to using that as the default?